### DISCUSSION TITLE
Add ability to create custom web parts that are "Made for email" compatible

### DISCUSSION THREAD

**Discussion by mbice** [SharePoint/sp-dev-docs#9900] [Ideas]
Hi,

We have noticed in the past few weeks that news posts are default filtered to "Made for email" web parts and if you have a template with any custom web parts on it (as most of our clients do) they get a warning at the top about web parts being incompatible with email.

Are there any plans to introduce the ability to configure a custom web part so that it can indeed be used in emails?

Or alternatively, give us the ability to create news post page templates that are marked as not email compatible so that authors (who don't need to send these posts as emails) don't see the web part toolbox filtered by default and no longer get the warning at the top if any custom web parts are on the page?

Thanks!

**Comment by GuillermoBas** [comment]
We are looking forward to this as well. Pretty frustrating for our users to not be able to share News content just because they have a simple custom authors web part next to the News content. We would love to enable (where it makes sense) web parts to be email compatible.
